core/code/portal_detail_display_tools.js

```diff
@@ -4,17 +4,17 @@
 
 window.getPortalHistoryDetails = function (d) {
   if (!d.history) {
-    return '<div style="text-align: center; color: #ffce00">History missing</div>';
+    return '<div id="history_details">History missing</div>';
   }
   var visited = d.history.visited;
   var captured = d.history.captured;
   var scoutControlled = d.history.scoutControlled;
-  var colors = {true:'#03fe03', false:'#ff4a4a'};
+  var className = {true:'done', false:''};
 
-  return '<div style="text-align: center; color: #ffce00">History: '
-  + '<span id="history_visited" style="color:'+ colors[visited] + '">visited</span> | '
-  + '<span id="history_captured" style="color:'+ colors[captured] + '">captured</span> | '
-  + '<span id="history_scout_controlled" style="color:' + colors[scoutControlled] + '">scout controlled</span>'
+  return '<div id="history_details">History: '
+  + '<span id="history_visited" class="'+ className[visited] + '">visited</span> | '
+  + '<span id="history_captured" class="'+ className[captured] + '">captured</span> | '
+  + '<span id="history_scout_controlled" class="' + className[scoutControlled] + '">scout controlled</span>'
   + '</div>';
```

Comment:
Perhaps this could be done with https://leafletjs.com/reference-1.7.1.html#util-template